packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/AppContext.kt

```diff
@@ -127,6 +130,16 @@ class AppContext(
     hostingRuntimeContext.registry.postOnCreate()
   }
 
+  fun registerLocalModulesListIfExists() {
+    try {
+      val localModulesList = Class.forName("local.modules.ExpoLocalModulesList").getConstructor()
+        .newInstance() as ModulesProvider
+      hostingRuntimeContext.registry.register(localModulesList)
+    } catch (e : Error) {
+      logger.error("Tried registering LocalModulesList but it doesn't exist: ", e.cause)
```

Comment:
```suggestion
```
We just want to ignore the error. Logging it will be incorrect if the local modules were disabled.